projects/components/src/lib/result-table/result-table.component.html

```diff
@@ -20,7 +20,7 @@
     <ng-container *ngIf="pagination">
       <ng-container *ngTemplateOutlet="pagination"></ng-container>
     </ng-container>
-    <div class="table-pagination" *ngIf="state === 'succeeded' && model && model.results.length > 0 && !pagination">
+    <div class="table-pagination" *ngIf="state === 'succeeded' && model && model.results.length > 0 && !pagination && !options?.hidePagination">
```

Comment:
Do we really need `!pagination`? Seems like it should be required unless they specify `options.hidePagination` as true. We should also check the options first, nitpick.